src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -572,8 +570,7 @@ public Color BorderlineColor
         /// </code>
         /// </para>
         /// </remarks>
-        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
please apply the same changes as to the ViewStyle BorderlineOffset

Reply:
Fix done.